README.rst

```diff
@@ -921,3 +948,30 @@ Running behind a proxy
 If you require a proxy to access Zyte API (e.g. a corporate proxy), configure
 the ``HTTP_PROXY`` and ``HTTPS_PROXY`` environment variables accordingly, and
 set the ``ZYTE_API_USE_ENV_PROXY`` setting to ``True``.
+
+
+Customizing the fallback download handler
+=========================================
+
+When the plugin needs to do requests directly, not via Zyte API, it uses a
+fallback download handler. It will use the Scrapy default one unless the
+``ZYTE_API_FALLBACK_HTTP_HANDLER`` and ``ZYTE_API_FALLBACK_HTTPS_HANDLER``
+settings are set. So if you want to use some other handler as a fallback, set
+those variables::
+
+    ZYTE_API_FALLBACK_HTTP_HANDLER = "myproject.MyHandler"
+    ZYTE_API_FALLBACK_HTTPS_HANDLER = "myproject.MyHandler"
+
+Alternatively, if you use the Scrapy addon, you can set the
```

Comment:
```suggestion
Alternatively, if you use the Scrapy addon, you can keep the
```

Reply:
If users are using cusotm downloade handlers, they already should be in their settings.